modules/calendar/calendar.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, id: str, name: str):
 		self.__name = name
 
 	@property
-	def id(self) -> int:
+	def calendar_id(self) -> str:
```

Comment:
I still disagree about not calling the property `id`.

It is unnecessarily redundant to have to always say `calendar.calendar_id` when you could simply say `calendar.id`.

Reply:
Could be... But I seem to remember trying it and something complaining, tho it might have just been pyre

Reply:
If you call something `id` in a global namespace or in a function, it will shadow the built-in and I've seen sonarlint make a warning for it.

If it's inside a class, though, it's a different namespace, so things like `self.id` or `calendar.id` can't collide with the built-in `id`.